bin/tx-prover/src/proxy/worker.rs

```diff
@@ -0,0 +1,102 @@
+// WORKER
+// ================================================================================================
```

Comment:
nit: we usually place section separators after imports (i.e., after line 13 in this file).